secant/Features/WalletEventsFlow/Views/TransactionDetailView.swift

```diff
@@ -112,33 +194,44 @@ extension TransactionDetailView {
     }
 }
 
+// MARK: - Row modifier
+
 struct TransactionDetailRow: ViewModifier {
-    let tint: Color
+    let mark: TransactionDetailView.RowMark
     let textColor: Color
     let backgroundColor: Color
 
     func body(content: Content) -> some View {
-        content
+        let markColor: Color
+        switch mark {
```

Comment:
I think this is oddly indented, isn't it?

Reply:
I pulled out the color switch to a private function which simplified the body of the modifier